conversion.c

```diff
@@ -83,23 +75,27 @@ char* Hexadecimal (int num) {
 	char* storeBin = Binary (num);                  // Stores the binary value of the string returned
```

Comment:
When you are trying to find the hexadecimal value of a number, you are again calculating the binary value. Instead as you have already calculated the binary value of the number you could store that value locally and pass it the function.